src/controllers/expenseController.js

```diff
@@ -0,0 +1,94 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filteredExpenses = await expensesService.getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
+
+  res.status(200).send(filteredExpenses);
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+  const expense = await expensesService.getExpenseById(id);
```

Comment:
In the `getOne` function, you are fetching the expense and then checking if it exists. If it doesn't exist, you send a 404 status code. This is correct, but you could improve the code by handling the case where the `getExpenseById` operation throws an error. For example, if the operation throws an error, you could catch that error and send a 500 Internal Server Error status code.